pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -259,6 +260,13 @@ func (a *Bootstrap) getTemplateData(installConfig *types.InstallConfig, releaseI
 		platformData.VSphere = vsphere.GetTemplateData(installConfig.Platform.VSphere)
 	}
 
+	// Set cluster profile
+	clusterProfile := ""
+	if cp := os.Getenv("OPENSHIFT_INSTALL_EXPERIMENTAL_CLUSTER_PROFILE"); cp != "" {
+		logrus.Warn("Found override for Cluster Profile. Please be warned.")
```

Comment:
Probably want to include `%q` for `cp` in this message with:

```go
logrus.Warnf("Found override for Cluster Profile: %q", cp)
```

No need for `Please be warned`; it's already a warning-level log, and logrus will set level=warning or whatever.

Reply:
fixed